vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -231,33 +231,33 @@ export default class SbcHeader extends NavigationMixin {
   logout () {
     if (this.redirectOnLogout) {
       const url = encodeURIComponent(this.redirectOnLogout)
-      window.location.assign(`${ConfigHelper.getAuthContextPath()}signout/${url}`)
+      window.location.assign(`${this.getContextPath()}signout/${url}`)
     } else {
-      window.location.assign(`${ConfigHelper.getAuthContextPath()}signout`)
+      window.location.assign(`${this.getContextPath()}signout`)
     }
   }
 
   login () {
     if (this.redirectOnLoginSuccess) {
       let url = encodeURIComponent(this.redirectOnLoginSuccess)
       url += this.redirectOnLoginFail ? `/${encodeURIComponent(this.redirectOnLoginFail)}` : ''
-      window.location.assign(`${ConfigHelper.getAuthContextPath()}signin/bcsc/${url}`)
+      window.location.assign(`${this.getContextPath()}signin/bcsc/${url}`)
     } else {
-      window.location.assign(`${ConfigHelper.getAuthContextPath()}signin/bcsc`)
+      window.location.assign(`${this.getContextPath()}signin/bcsc`)
```

Comment:
It's not clear to me what this change is doing?  Also, why did these URLs need to be changed?

Reply:
This is done for getting the context path according to the parent application using our components. so that the signin and signout will be loosely coupled